File: Tamago-FrontEnd/src/app/Models/InventoryModel.ts
export class InventoryModel {
  public id: String;
  public userId: String;

  public itemId: String[];

  constructor(id: String, userId: String, itemId: String[]) {
    this.id = id;
    this.userId = userId;
    this.itemId = itemId;
  }
}

File: Shared/ApiModels/InventoryModel.cs
using System;
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;
using System.Linq;
using System.Threading.Tasks;

namespace Shared.Shared.ApiModels
{
    [ExcludeFromCodeCoverage]
	public class InventoryModel
	{ 
		public Guid id { get; set; }
		public Guid userId { get; set; }
		public List<Guid> itemId { get; set; }
	}
}
